packages/im-ott/mock.json

```diff
@@ -0,0 +1,172 @@
+[
```

Comment:
The file should be mockData.json or mockMovieData.json to be more meaningful. We can later use the same file in test cases as well